projects/packages/jetpack-mu-wpcom/src/features/block-theme-previews/block-theme-previews.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Gutenberg's Block Theme Previews feature
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+/**
+ * Always show the correct homepage when previewing a theme in the Site Editor
+ *
+ * @see https://github.com/Automattic/wp-calypso/issues/79221
+ * @since 12.4
+ */
+add_filter(
```

Comment:
Maybe we can add the filter or even load this feature only when the `wp_theme_preview` is set 🤔

Reply:
Addressed in https://github.com/Automattic/jetpack/pull/31929/commits/3800d38442b703333f764cf5559a17c0d9d4ddae!